src/gui/instrument/InstrumentMidiIOView.cpp

```diff
@@ -144,6 +146,20 @@ InstrumentMidiIOView::InstrumentMidiIOView( QWidget* parent ) :
 		midiOutputLayout->insertWidget( 0, m_wpBtn );
 	}
 
+	m_presetSelectGroupBox = new GroupBox(tr("ENABLE PRESET SELECTION"));
+	layout->addWidget( m_presetSelectGroupBox );
+
+	QHBoxLayout * presetSelectLayout = new QHBoxLayout(m_presetSelectGroupBox);
+	presetSelectLayout->setContentsMargins( 8, 18, 8, 8 );
+	presetSelectLayout->setSpacing( 6 );
+	m_presetSelectPolicyComboBox->setFixedSize(QSize(220, 20));
+	m_presetSelectPolicyComboBox->setEnabled(false);
+
+	presetSelectLayout->addWidget(m_presetSelectPolicyComboBox);
+	presetSelectLayout->addStretch();
+
+	connect( m_presetSelectGroupBox->ledButton(), SIGNAL( toggled( bool ) ),
+			m_presetSelectPolicyComboBox, SLOT( setEnabled( bool ) ) );
```

Comment:
I would rather we switch this to new Qt style connect.

Reply:
done